test/bn_vectors.js

```diff
@@ -76,6 +76,9 @@ sjcl.test.vector.bn_powermod = [
   // // This was disabled because it's slow (~2.5s with `rhino -O 9`). Once
   // // a better powermod algorithm is implemented (e.g., using Montgomery
   // // reduction), this can be re-enabled.
+  //
+  // // Note:  This example can't be done with Montgomery reduction anyways
```

Comment:
As we are now on nodejs, the old tests could also be re-added (it should be fast enough)